reVX/config/transmission_layer_creation.py

```diff
@@ -80,13 +82,56 @@ class WetCosts(BaseModel, extra='forbid'):
     bathy_tiff: FilePath
     """File name of bathymetric depth GeoTIFF"""
 
-    wet_costs_tiff: Optional[str] = None
-    """Name for wet costs GeoTIFF"""
-
 
 class DryCosts(BaseModel, extra='forbid'):
     """ Config items required to generate dry costs """
-    # TODO
+    iso_region_tiff: FilePath
+    """Filename of ISO region GeoTIFF"""
+
+    nlcd_tiff: FilePath
+    """File name of NLCD GeoTiff"""
+
+    slope_tiff: FilePath
+    """File name of slope GeoTiff"""
+
+    cost_configs: Optional[FilePath] = None
+    """Path to json file with Xmission cost configuration values.
+
+    Path to json file contianing dictionary with Xmission cost
+    configuration values. Valid configuration keysare:
+
+        - "base_line_costs"
+        - "iso_lookup"
+        - "iso_multipliers"
+        - "land_use_classes"
+        - "new_substation_costs"
+        - "power_classes"
+        - "power_to_voltage"
+        - "transformer_costs"
+        - "upgrade_substation_costs"
+
+    Each of these keys should point to a dictionary or a path to
+    a separate json file contianing a dictionary of
+    configurations for each section."""
+
+    default_mults: Optional[IsoMultipliers] = None
+    """Multipliers to be used for default region.
+
+    This input should be a dictionary with three keys:
+
+        - "iso": Thie key is ignored, but is required. Can set to
```

Comment:
A required but ignored key, that sounds like I screwed up. Can we pull it entirely?

Reply:
Just looked and I think we can! I wasn't sure if you had some purpose for it or not so that's why i left it, but I will go ahead and remove it now. Good catch!